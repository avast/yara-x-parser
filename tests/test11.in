rule test : tag1 tag2
{
	strings:
		$a = "foo"
	condition:
		$a
}