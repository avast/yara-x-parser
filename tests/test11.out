SOURCE_FILE@0..65
  RULE@0..65
    RULE_KW@0..4 "rule"
    WHITESPACE@4..5 " "
    IDENTIFIER@5..9 "test"
    WHITESPACE@9..10 " "
    COLON@10..11 ":"
    WHITESPACE@11..12 " "
    TAG@12..16
      IDENTIFIER@12..16 "tag1"
    WHITESPACE@16..17 " "
    TAG@17..21
      IDENTIFIER@17..21 "tag2"
    WHITESPACE@21..22 "\n"
    BLOCK_EXPR@22..65
      L_BRACE@22..23 "{"
      WHITESPACE@23..25 "\n\t"
      STRINGS@25..46
        STRINGS_KW@25..32 "strings"
        COLON@32..33 ":"
        WHITESPACE@33..36 "\n\t\t"
        VARIABLE_STMT@36..46
          VARIABLE@36..38 "$a"
          WHITESPACE@38..39 " "
          ASSIGN@39..40 "="
          WHITESPACE@40..41 " "
          PATTERN@41..46
            STRING_LIT@41..46 "\"foo\""
      WHITESPACE@46..48 "\n\t"
      CONDITION@48..63
        CONDITION_KW@48..57 "condition"
        COLON@57..58 ":"
        WHITESPACE@58..61 "\n\t\t"
        EXPRESSION_STMT@61..63
          BOOLEAN_TERM@61..63
            VARIABLE@61..63 "$a"
      WHITESPACE@63..64 "\n"
      R_BRACE@64..65 "}"
